components/Modal/Modal.jsx

```diff
@@ -1,34 +1,32 @@
 import { FontAwesomeIcon } from "@fortawesome/react-native-fontawesome";
 import { faXmark } from "@fortawesome/free-solid-svg-icons";
-import { TouchableOpacity, View, ScrollView } from "react-native";
+import { TouchableOpacity, View, TouchableWithoutFeedback, Keyboard } from "react-native";
 import { ModalContext } from "./ModalProvider";
 import { useContext } from "react";
 
 export default function Modal() {
     const { modal, setModal } = useContext(ModalContext);
     if (modal != null) {
         return (
-            <View className="flex flex-1 h-screen justify-center items-center bg-[#FFFFFF90]">
-                <View className={`w-10/12 h-fit pb-4 border-2 rounded-xl  bg-white`}>
-                    <ScrollView
-                        className="flex flex-col"
-                        scrollEnabled={false}
-                        keyboardShouldPersistTaps="handled"
-                    >
-                        <View className="flex-row">
-                            <TouchableOpacity
-                                onPress={() => {
-                                    setModal(null);
-                                }}
-                                className="w-12 h-12 flex justify-center items-center"
-                            >
-                                <FontAwesomeIcon icon={faXmark} size={30} />
-                            </TouchableOpacity>
+            <TouchableWithoutFeedback onPress={() => setModal(null)}>
+                <View className="h-screen justify-center items-center bg-[#FFFFFF90]">
```

Comment:
Can you not make this View a Touchable without feedback?

Reply:
and not wrap it with one?

Reply:
done. had to use Pressable instead cos TouchableWithoutFeedback cant have multiple children and cant take a classname but defo much cleaner now